src/components/Modals/AddContactModal.jsx

```diff
@@ -40,7 +40,6 @@ const renderWebId = (username) => {
  * @param {Function} props.addContact - Function to add a contact
  * @param {boolean} props.showAddContactModal - Whether to display modal or not
  * @param {Function} props.setShowAddContactModal - Toggle modal
- * @returns {React.JSX.Element} - The Add Contact Modal
```

Comment:
It looks like JSDoc return statement has been removed by mistake as pointed out by the linter.

```
* @returns {React.JSX.Element} - The Add Contact Modal
```